artifactory/services/delete.go

```diff
@@ -216,11 +215,17 @@ func removeNotToBeDeletedDirs(specFile *utils.CommonParams, ds *DeleteService, d
 		if err != nil {
 			return nil, err
 		}
-		artifactNotToBeDeleteReader, err := getSortedArtifactsToNotDelete(specFile, ds)
+		var artifactNotToBeDeleteReader *content.ContentReader
+		artifactNotToBeDeleteReader, err = getSortedArtifactsToNotDelete(specFile, ds)
 		if err != nil {
 			return nil, err
 		}
-		defer artifactNotToBeDeleteReader.Close()
+		defer func() {
+			e := artifactNotToBeDeleteReader.Close()
+			if err == nil {
+				err = errorutils.CheckError(e)
```

Comment:
This is the close implementation of our ContentReader. I don't think the check error here is necessary.